static/style.css

```diff
@@ -128,7 +145,12 @@ input[type="text"]:focus {
  button:hover {
    background: #49afff;
  }
-
+ 
+@media screen and (min-width: 767px) {
```

Comment:
Please combine all `@media` selectors into one at the end of the file. You can re-define styles if necessary. That way the style for mobile devices can be adjusted without changing the desktop version.